plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftPvcHelper.java

```diff
@@ -176,7 +177,7 @@ protected boolean createJobPod(String workspacesPvcName,
         try (OpenShiftClient openShiftClient = new DefaultOpenShiftClient()){
             openShiftClient.pods().inNamespace(projectNamespace).create(podSpec);
             boolean completed = false;
-            while(!completed) {
+            for(int waitCount = 0; (waitCount < JOB_TIMEOUT_SECONDS) && !completed; waitCount++) {
```

Comment:
Yeah I took a second look at this loop and it really made no sense -- completed was never set to true...

Fixed to wait in the same was as OpenShiftConnector.